internal/auth0/organization/resource_connection.go

```diff
@@ -38,9 +39,24 @@ func NewConnectionResource() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 				Default:  false,
-				Description: "When true, all users that log in with this connection will be automatically granted " +
-					"membership in the organization. When false, users must be granted membership in the organization" +
-					" before logging in with this connection.",
+				Description: "When `true`, all users that log in with this connection will be automatically granted " +
+					"membership in the organization. When `false`, users must be granted membership in the organization " +
+					"before logging in with this connection.",
+			},
+			"is_signup_enabled": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Default:  false,
```

Comment:
If we're getting this through the `value` package (https://github.com/auth0/terraform-provider-auth0/pull/974/files#diff-4e18de8617087e7ec36d59473c8642069274c15a5a5b2fdd041a2ba817a3453aR83), then the default value won't get picked up, but we will display it within the docs erroneously. 

So maybe let's not have any default values for these 2 fields.

Reply:
if we get rid of the default here, the plan breaks. It appears to be a bug in SDK v2

Reply:
Ah I see now, the plan will detect that this was there before and if we remove it from the config it will think it needs to set it as null, causing a diff after the apply. 

That sounds like we should set this as computed and not provide a default tho. Otherwise we risk changing this to an unexpected value.